### ISSUE TITLE
Wrong ticket limit enforced on checkout

### ISSUE THREAD

**Issue by GijsWeterings** [WISVCH/events#186]
If I add 21 tickets of option 1 (see also #185) and go to the checkout process, I get the following error box:

![](https://i.imgur.com/ZlM4Pua.png)

1. This is not an error, as I still can add more according to the message
2. I tried to buy zero tickets for Option 2, but went over the limit for Option 1
3. The pluralization in the template doesn't really make sense with this item name, maybe rephrase it?

**Comment by Fastjur** [comment]
I think the error message is correct. You probably already have 17 tickets in total, thus you are allowed to only buy 3 more.

**Comment by GijsWeterings** [comment]
I had 21 tickets of option 1 and zero of option 2. In that case, It is indeed true I can only buy 3 more of option 2, as I didn't order any at all. The error message should have warned me I tried to buy too many tickets of option 1, however that error was swallowed (and my shopping cart cleared)

**Comment by Fastjur** [comment]
I see. I understand the issue then.
